core/arch/proc_api.h

```diff
@@ -175,6 +180,17 @@ typedef struct {
 } features_t;
 typedef enum { AA64ISAR0 = 0, AA64ISAR1 = 1, AA64PFR0 = 2 } feature_reg_idx_t;
 #endif
+#ifdef RISCV64
+/* FIXME-RISCV: Not implemented */
+/**
+ * For RISC-V64 there are no features readable from userspace. Hence only a
+ * dummy flag is there. May be replaced by actual feature flags in the future.
+ * Used by proc_get_all_feature_bits().
+ */
+typedef struct {
+    uint64 dummy; /**< dummy member to keep size non-0. */
```

Comment:
nit: Prefer capitalized sentences in comments, in particular in user-facing Doxygen comments: `s/dummy/Dummy/`.

Reply:
Done.